configurations/configurations.go

```diff
@@ -0,0 +1,61 @@
+package configurations
+
+import (
+	"encoding/json"
+	"errors"
+	"io/ioutil"
+	"log"
+	"os"
+)
+
+type Configuration struct {
+	MaxMRUDisplay string `json:"MaxMRUDisplay"`
+}
+
+func DefaultConfigurations() Configuration {
+	return Configuration{
+		MaxMRUDisplay: "10",
```

Comment:
Probably want to have `int` type for this?

Reply:
Fixed.